person.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'nameable'
+class Person < Nameable
+  attr_accessor :id, :name, :age
```

Comment:
Great job @haftamudesta 🤝

- [ ] Kindly ensure the Getters and Setters for @rentals are present.